beaglecli

```diff
@@ -351,6 +363,33 @@ def _get_latest_runs(run_dict):
     return run_list
 
 
+def _get_cwl_apps():
+    url = urljoin(BEAGLE_ENDPOINT, API['pipelines'])
+    params = dict()
+    params['page_size'] = 1000000
```

Comment:
I've seen some slow downs / bad responses when setting this parameter too high for the files system endpoint. I don't think that should be an issue here since the pipelines list is relatively short. Nonetheless would we maybe want to make this an adjustable parameter somewhere?  Magic numbers like these can sometimes be a bit tricky to track down when there's an issue.